website/common/locales/en/groups.json

```diff
@@ -415,5 +415,6 @@
   "wouldYouParticipate": "Would you like to participate?",
   "managerAdded": "Manager added successfully",
   "managerRemoved": "Manager removed successfully",
-  "leaderChanged": "Leader has been changed"
+  "leaderChanged": "Leader has been changed",
+  "groupNoNotifications": "This Guild does not have notifications due to member size. Be sure to check back often for replies to your messages!"
```

Comment:
there's the string `  "largeGroupNote": "Note: This Guild is now too large to support notifications! Be sure to check back every day to see new messages.",
`

Reply:
Okay, I'll remove the old string as the string I've added is from Zeplin.